sops/migration/README.md

```diff
@@ -0,0 +1,64 @@
+# Action
+If you are currently using apps.ci.centos.org (OCP 3.6) or have a job running in ci.centos.org, please contact us so that we can create a namespace for you in the new OCP 4.4 cluster for you to start migrating your jobs.
+
+
+## If your own a namespace on apps.ci.centos.org
+There shouldn't be a lot of changes for you as you should already have a familiar workflow working with Openshift, if you have configured your jobs at least once. While opening a ticket, or reaching out to us please do mention if you need ability to checkout duffy nodes (if yes, do you already have a duffy API key?).
+
+See the following which explains how to open a ticket (Point 3)
+
+
+## If your project is on ci.centos.org
+As you are using central jenkins deployment that's not private to you, your project might see some changes. You will have your own jenkins deployment to run all your jobs instead of having a shared jenkins instance where you don't have admin access. With the extra privilage in jenkins, you can add plugins, credentials as you need and a bunch of other abilities.
+This also means there is no direct maintenance required from CentOS CI Infra admins as openshift takes care of things dying or crashing and anything in the jobs config, you can take care of them yourself (We will help you get started and are always in reach if you notice any hiccups that is not auto healed). You can have the same workflow or use this opportunity to change your freestyle jobs to pipelines but that's totally on you.
+See the following which explains how to open a ticket (Point 3)
+
+
+### open a project ticket
+
+* login to bugs.centos.org
+* Select CentOS CI project
+* Create an issue with following info (You can also comment in a different private comment if you prefer your email addresses to be private)
+
+```
+namespace name:
+namespace admins
+ - user1@email.com // email address used in accounts.centos.org
+ - user2@email.com // email address used in accounts.centos.org
+
+Do you use baremetal nodes for your jobs?: Yes/No
+```
+* Assign it to `siddharthvipul1` or `dkirwan`
```

Comment:
I'd say just leave that part out, as being created in the right location is enough. Whoever is working on resolving the tickets, will pick it up when they are free.

Reply:
yeah, if we end up going for pagure.io, we can eliminate the whole thing with just one line.
Waiting till we have finalized that